aip/general/0008.md

```diff
@@ -191,12 +209,32 @@ subdirectory); this ensures that the link works both on the AIP site, when
 viewing the Markdown file on GitHub, using the local development server, or a
 branch.
 
+## Rationale
+
+### Designing for a broad set of clients
+
+API guidance, similar to any software, is most beneficial when there is a clear
+purpose and target beneficiary.
+
+The beneficiaries of improved API design are users. These users interact with
+APIs via a variety of methods, depending on their use case:
```

Comment:
via a variety of clients, since that's the topic here

"methods" could also be interpreted as rpcs

Reply:
ack, good call.